src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -441,6 +441,10 @@ public string TargetedRuntimeVersion
             set { _targetedRuntimeVersionRawValue = value; }
         }
 
+        public string PreComputedCacheOutputPath { get; set; }
+
+        public string[] PreComputedCacheFileList { get; set; }
```

Comment:
```suggestion
        public string[] AssemblyInformationCache { get; set; }
```

?

Reply:
"AssemblyInformation" just sounds like "related to RAR" to me. It would probably be best to match this with the above name, so the equivalents of my latest suggestions would be (Secondary)CacheInputPath, Relative(Path)CacheInputPath, or InputCachePath. Do any of those sound reasonable to you?